README.adoc

```diff
@@ -1,7 +1,39 @@
 = Connect Account
+:toc: preamble
 
+Connect Account is an open source app, which aims at providing all the business logic to handle Connect identities and profile data, and a default frontend for User to use it.
 
-## Setup
+== Flows
+
+=== Connect authentication flow
+
+image::./doc/oauth2-flow-diagram.svg[OAuth2 flow diagram]
+
+For more information, check the Authorization code flow https://developers.fewlines.co/guides/authorization_code_flow/overview/[documention].
```

Comment:
```suggestion
For more information, check the Authorization code flow https://developers.fewlines.co/guides/authorization_code_flow/overview/[documentation].
```

Reply:
Please, don't tell me that this typo is the real URL. 😅

Reply:
Haha no, it's just the way `asciidoc` handle hyperlinks.

Reply:
a62d1f2 Fixed. If you could re-approved, thanks :).